frontend/src/properties/propertysections/TextfieldPropertySection.tsx

```diff
@@ -83,6 +72,18 @@ export const TextfieldPropertySection = ({ projectId, formId, widgetId, label, v
     await updateWidgetFocus({ variables });
   };
 
+  useEffect(() => {
+    console.log(editTextFieldResult);
+    if (editTextFieldResult.loading) {
+      console.log('HANDLE_SUBMIT');
+      dispatch({ type: 'HANDLE_SUBMIT' });
+    } else if (editTextFieldResult.called && editTextFieldResult.data?.editTextfield) {
+      console.log('HANDLE_RESPONSE');
+      editTextFieldResult.called = false;
+      dispatch({ type: 'HANDLE_RESPONSE' });
+    }
+  }, [editTextFieldResult, dispatch]);
```

Comment:
to rework
How to not call   dispatch({ type: 'HANDLE_RESPONSE' }); at each react TextfieldPropertySection refresh after the apollo response has been returned